src/shogun2-util/src/main/java/de/terrestris/shogun2/util/interceptor/OgcNaming.java

```diff
@@ -0,0 +1,101 @@
+package de.terrestris.shogun2.util.interceptor;
+
+/**
+ *
+ * @author Daniel Koch
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+public class OgcNaming {
+
```

Comment:
I think we should get rid of this class and use Enums instead (see comments above)